rflx/expression.py

```diff
@@ -733,6 +728,10 @@ def __neg__(self) -> Expr:
     def precedence(self) -> Precedence:
         return Precedence.literal
 
+    @abstractproperty
```

Comment:
We decided to switch to
```Python
@property
@abstractmethod
```
in #189. Should we change other uses of `@abstractproperty` each time we touch them?

Reply:
I would prefer to fix them all at once. I created issue for this purpose (#204).

Reply:
This sounds good to me.